### DISCUSSION TITLE
Looking to get a v1.2/3 board

### DISCUSSION THREAD

**Discussion by mattenz** [Arnold-n/P1P2MQTT#76] [General]
Hi there,

I've got a Daikin FDYAN125AV1 being installed and I came across this project as a means to control it via Home Assistant. I followed the Readme and sent an email to the address found elsewhere to go about getting a board but it's been a couple weeks and I haven't heard back so I'm wondering if I may have gotten the address wrong. I had also looked for a schematic so I could look at building one myself but it seems only the older version is available.

**Comment by Arnold-n** [comment]
Hi @mattenz,
Sorry I overlooked your mail, you had the address correct, and I just replied to you.
